x-pack/legacy/plugins/maps/public/shared/layers/sources/ems_tms_source/create_source_editor.js

```diff
@@ -23,6 +24,12 @@ export class EMSTMSCreateSourceEditor extends React.Component {
 
   _loadTmsOptions = async () => {
     const options = await getEmsTMSServices();
+    options.push({
```

Comment:
I agree that we can preserve auto-option as just one options in the dropdown. (the wizard just needs to make sure it creates the right descriptor. See above comment on shape of descriptor)

Reply:
Add auto-mode as first option to the list, instead of last

Instead of `Auto` I would call it something like `Autoselect based on Kibana theme`